smoke-tests-otel-starter/src/test/java/io/opentelemetry/smoketest/OtelSpringStarterSmokeTest.java

```diff
@@ -53,16 +53,13 @@
     },
     webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
     properties = {
-      "otel.traces.exporter=none",
-      "otel.metrics.exporter=none",
-      "otel.logs.exporter=none",
       "otel.metric.export.interval=100",
+      // We set the export interval of the metrics to 100 ms. The default value is 1 minute.
+      // the headers are simply set here to make sure that headers can be parsed
```

Comment:
(just moving the comment above instead of below the line of code)
```suggestion
      // We set the export interval of the metrics to 100 ms. The default value is 1 minute.
      // the headers are simply set here to make sure that headers can be parsed
      "otel.metric.export.interval=100",
```